src/ApiService/ApiService/ServiceConfiguration.cs

```diff
@@ -82,6 +83,8 @@ public class ServiceConfiguration : IServiceConfig {
     public string? ApplicationInsightsAppId => GetEnv("APPINSIGHTS_APPID");
     public string? ApplicationInsightsInstrumentationKey => GetEnv("APPINSIGHTS_INSTRUMENTATIONKEY");
 
+    public string? AppConfigurationConnectionString => GetEnv("APPCONFIGURATION_CONNECTION_STRING");
```

Comment:
And so this is not needed when using rbac with managed identity

Reply:
but you still need to pass feature flags info in order to instantiate the object and then use 

new Azure.Identity.DefaultAzureCredential(false)

for auth